### PR TITLE
bump qtconsole version requirement to fix #1854

### LINKED ISSUE #1854: Tab completion in QT console causes float divide by zero error
## 🐛 Bug

When trying do to tab completion in the Qt console the current master, I get a float divide by zero error popping up.

tab_completion

In a slightly earlier version (not sure which one I had installed before pulling the latest master) there was no error message but napari crashed with the following traceback. 
This suggests it may be an upstream error in qtconsole.

```
(napari) volker@gpubox:~$ napari
WARNING: Traceback (most recent call last):
  File "/home/volker/miniconda3/envs/napari/lib/python3.7/site-packages/qtconsole/base_frontend_mixin.py", line 138, in _dispatch
    handler(msg)
  File "/home/volker/miniconda3/envs/napari/lib/python3.7/site-packages/qtconsole/jupyter_widget.py", line 178, in _handle_complete_reply
    self._complete_with_items(cursor, matches)
  File "/home/volker/miniconda3/envs/napari/lib/python3.7/site-packages/qtconsole/console_widget.py", line 1054, in _complete_with_items
    prefix_length=len(prefix))
  File "/home/volker/miniconda3/envs/napari/lib/python3.7/site-packages/qtconsole/completion_html.py", line 313, in show_items
    displaywidth = int(max(10, (width / char_width) - 1))
ZeroDivisionError: float division by zero
10:24:13 WARNING Traceback (most recent call last):
  File "/home/volker/miniconda3/envs/napari/lib/python3.7/site-packages/qtconsole/base_frontend_mixin.py", line 138, in _dispatch
    handler(msg)
  File "/home/volker/miniconda3/envs/napari/lib/python3.7/site-packages/qtconsole/jupyter_widget.py", line 178, in _handle_complete_reply
    self._complete_with_items(cursor, matches)
  File "/home/volker/miniconda3/envs/napari/lib/python3.7/site-packages/qtconsole/console_widget.py", line 1054, in _complete_with_items
    prefix_length=len(prefix))
  File "/home/volker/miniconda3/envs/napari/lib/python3.7/site-packages/qtconsole/completion_html.py", line 313, in show_items
    displaywidth = int(max(10, (width / char_width) - 1))
ZeroDivisionError: float division by zero
```

## To Reproduce

1. start napari
2. open ipython console
3. type viewer.
4. press TAB

## Expected behavior

Possible completions shown, no error message.

## Environment

```
napari: 0.4.1a1
Platform: Linux-5.4.0-52-generic-x86_64-with-debian-bullseye-sid
Python: 3.7.7 (default, May 7 2020, 21:25:33) [GCC 7.3.0]
Qt: 5.14.2
PyQt5: 5.14.2
NumPy: 1.19.1
SciPy: 1.5.2
Dask: 2.25.0
VisPy: 0.6.4

GL version: 4.6.0 NVIDIA 450.80.02
MAX_TEXTURE_SIZE: 32768

Plugins:
- napari_plugin_engine: 0.1.8
- svg: 0.1.4
```

likely relevant:

image

### PR DESCRIPTION
**Pull request by VolkerH**
# Description

Due to a bug in a recent version of qtconsole, TAB-completion in the console was broken #1854.
This bumps the requirement for qtconsole up to version 4.7.7 in which the bug is fixed.

## Type of change
- [x] Bug-fix (non-breaking change which fixes an issue)

# References
#1854 

# How has this been tested?
pip-installed napari in an environment that did not have qtconsole.
verified manually that installed version is 4.7.7 and tab completion works.

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -52,7 +52,7 @@ install_requires =
     psutil>=5.0
     PyOpenGL>=3.1.0
     PyYAML>=5.1
-    qtconsole>=4.5.1
+    qtconsole>=4.7.7
```
Comment (jni) [inline setup.cfg]: @VolkerH could this be resolved by blocking a specific version?

```suggestion
    qtconsole>=4.5.1,!=4.7.6
```

Comment (VolkerH) [reply setup.cfg]: probably... are there any benefits in supporting old qtconsole versions though? Given that napari has already dropped python 3.6 and generally uses recent versions of libraries I don't really see a benefit.

Comment (VolkerH) [reply setup.cfg]: But if you think otherwise, I'm happy to commit your suggestion.

Comment (sofroniewn) [reply setup.cfg]: I just checked `4.5.1`, `4.7.5` and `4.7.7` all work, but `4.7.6` does not. Given that the bug seems localized to just that version I'd follow the recommendation of @jni here and just block that specific version as it won't force users to upgrade unnecessarily

Comment (VolkerH) [reply setup.cfg]: Thanks @jni, @sofroniewn 
Convinced. Change applied.

**Review by sofroniewn — APPROVED** [review]
LGTM!

**Comment by VolkerH** [comment]
Thanks!

**Pull request merged**